test/test_package.py

```diff
@@ -112,7 +112,7 @@ def test_init_kwargs_string(self):
 
     def test_init_kwargs_object(self):
         mmain = [self.get_maintainer(), self.get_maintainer()]
-        mlis = ['MIT', 'BSD']
+        mlis = [Mock(), Mock()]
```

Comment:
Please use `License` instances here and check for their value.

Reply:
Done.